src/androidTest/java/androidx/core/view/MenuTest.kt

```diff
@@ -132,4 +146,17 @@ class MenuTest {
         assertFalse(item2 in menu)
         assertEquals(0, menu.size())
     }
+
+    @Test fun children() {
+        val items = listOf(
+            menu.add(NONE, 1, NONE, ""),
+            menu.add(NONE, 2, NONE, ""),
+            menu.add(NONE, 3, NONE, "")
+        )
+        items.forEach { menu.add(it.groupId, it.itemId, it.order, it.title) }
```

Comment:
Is there any reason for doing this?

Reply:
Oops, sorry, didn't notice the the items have already been added upon filling list up.